src/components/productlist/productlist.component.html

```diff
@@ -0,0 +1,7 @@
+<div>
+    <div class="row">
+        <div class="col-sm" *ngFor="let product of products">
```

Comment:
Is this responsive to having many products? I know we only have 3 in our test data and it renders fine, but I wonder how it would look with more.

Reply:
I was on another planet when i did this. honestly, I dont remember actually doing it.

Reply:
I'm debating abandoning this branch and going back to main..